### PR TITLE
Add caption for free to free busy modal

### LINKED ISSUE #3113: Add label for "free" state in Free/Busy overview
Add an additional label for the "free" state in the overview to clarify that an attendee/resource is free when their row has a white background colour.

Mockup:

image

### PR DESCRIPTION
**Pull request by st3iny**
Fixes #3113

Screenshot 2021-06-08 at 08-25-41 June 2021 - Calendar - Nextcloud

### CODE REVIEW

File: src/components/Editor/FreeBusy/FreeBusy.vue [hunk 1]
```diff
@@ -196,6 +196,9 @@ export default {
 		 */
 		colorCaption() {
 			return [{
+				label: this.$t('calendar', 'Free'),
+				color: getColorForFBType('FREE'),
+			},{
```
Comment (tcitworld) [inline src/components/Editor/FreeBusy/FreeBusy.vue:201]: Lint issue.

File: src/components/Editor/FreeBusy/FreeBusy.vue [hunk 2]
```diff
@@ -196,6 +196,9 @@ export default {
 		 */
 		colorCaption() {
 			return [{
+				label: this.$t('calendar', 'Free'),
```
Comment (tcitworld) [inline src/components/Editor/FreeBusy/FreeBusy.vue:199]: Maybe add a `// TRANSLATORS` comment above to indicate how to translate `Free` (as in available).

**Review by tcitworld — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
Works, also with dark theme :+1:

File: src/components/Editor/FreeBusy/FreeBusy.vue (hunk 2 above)
Comment (st3iny) [reply src/components/Editor/FreeBusy/FreeBusy.vue:200]: Done

File: src/components/Editor/FreeBusy/FreeBusy.vue (hunk 1 above)
Comment (st3iny) [reply src/components/Editor/FreeBusy/FreeBusy.vue]: Done

**Pull request merged**